.storybook/stories/DataFiltering/lib/useFetchData.ts

```diff
@@ -28,6 +28,7 @@ export function useFetchData({
 	const [totalItems, setTotalItems] = useState(0);
 
 	useEffect(() => {
+		setLoading(true);
```

Comment:
While technically correct, the skeleton loading state is so egregious, I wonder if it was truly only meant for that initial render, and not every page/filter change? Probably a bigger question for the design team…?

Reply:
Hmm yeah, good question. I just figured you'd always want to ensure up-to-date data whenever you perform any sort of navigation within the application. We should jam on this, for sure.

- [ ] Team jam re: pagination + data sync